R/fda-table_8.R

```diff
@@ -0,0 +1,56 @@
+# Pre-processing
+library(dplyr)
+library(scda)
+library(tern)
+library(rlistings)
+
+# Deaths
+adae <-
+  synthetic_cdisc_dataset("rcd_2022_06_27", "adae") %>%
+  filter(SAFFL == "Y" & AESDTH == "Y") %>%
+  select(
+    STUDYID, USUBJID, ACTARM, ACTARMCD, AGE, SEX, AETERM, AEDECOD, ASTDY, AESDTH, AEOUT, DTHADY
```

Comment:
I'm not entirely sure what these variables should be, but since they are "Cause of Death" variables, I would change them to `DTHCAT` and `DTHCAUS` for now as that's probably somewhat more accurate.
```suggestion
    STUDYID, USUBJID, ACTARM, ACTARMCD, AGE, SEX, DTHCAT, DTHCAUS, ASTDY, AESDTH, AEOUT, DTHADY
```